README.md

```diff
@@ -13,7 +10,7 @@ Add `wdio-browserstack-reporter` as a dependency in your `package.json`.
 ```json
 {
   "dependencies": {
-    "wdio-browserstack-reporter": "~0.1.0"
+    "wdio-browserstack-reporter": "browserstack/wdio-browserstack-reporter#wdio5"
```

Comment:
this should be npm package now because it is in master and about to be published?

Reply:
Why arent we pointing it to 1.0.0 itself?